src/js/components/Articles.js

```diff
@@ -51,4 +51,4 @@ export default class Articles extends Component {
       </div>
     )
   }
-}
\ No newline at end of file
+}
```

Comment:
Yura, configure your editor to NOT automatically add new lines at the end of the document. Is it hard to just copy config from gist? It's third time I'm asking :(